ops/terraform/remote_files/scripts/install-node.sh

```diff
@@ -8,7 +8,7 @@ source /terraform_node/variables
 function install-go() {
   echo "Installing Go..."
   rm -fr /usr/local/go /usr/local/bin/go
-  curl --silent --show-error --location --fail https://go.dev/dl/go1.21.8.linux-amd64.tar.gz | sudo tar --extract --gzip --file=- --directory=/usr/local
+  curl --silent --show-error --location --fail https://go.dev/dl/go1.23.0.linux-amd64.tar.gz | sudo tar --extract --gzip --file=- --directory=/usr/local
```

Comment:
please deploy to staging to verify all is working as expected

Reply:
Done. working as intended on this commit https://github.com/bacalhau-project/bacalhau/pull/4332/commits/7404e375d09be15015438a2f10d6a17d18b83783:
```
export BACALHAU_API_HOST=bootstrap.staging.bacalhau.org
~> bacalhau agent version
Bacalhau v1.4.1-rc3-28-g7404e375d
BuildDate 2024-08-18 01:42:44 +0000 UTC
GitCommit 7404e375d09be15015438a2f10d6a17d18b83783

```